src/codegen/property_body.rs

```diff
@@ -73,9 +78,12 @@ impl Builder {
         let mut params = Vec::new();
 
         let cast_target = if self.is_child_property {
-            "gtk_sys::GtkContainer"
+            "ffi::GtkContainer".to_owned()
```

Comment:
If there are ever child properties in other crates (libhandy and libdazzle seem good candidates), this would have to become `gtk::ffi` (vs. now `crate::ffi`). So maybe add a case for this already

Reply:
Also elsewhere below

Reply:
Ok!